docs/guides/integration-prebid.md

```diff
@@ -1,172 +1,218 @@
 
-## UID2 User ID Submodule
+Once you have added Prebid.js to your site and confirmed that it is working properly, you are ready to configure the UID2 module.
 
-UID2 requires initial tokens to be generated server-side. The UID2 module handles storing, providing, and optionally refreshing them. The module operates in Client Refresh mode.
+:::tip
+Ensure the UID2 module is installed by finding the string "uid2IdSystem" in the [`pbjs.installedModules` array](https://docs.prebid.org/dev-docs/publisher-api-reference/installedModules.html).
+:::
 
->**Important:** UID2 is not designed to be used where GDPR applies. The module checks the passed-in consent data, and does not operate if the `gdprApplies` flag is set to `true`.
+## Configure the UID2 module
 
-## Client Refresh Mode
+To configure the UID2 module, call `pbjs.setConfig` with an object containing the **public key** and **subscription ID** obtained during account setup, as well as the user's [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number).
 
-You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+Once configured, the UID2 module will generate a UID2 token for the user and store it in the user's browser. The module will automatically refresh the token as required while your site is open in the user's browser.
 
-- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
-- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+When the UID2 module is configured it will check the user's browser for an existing UID2 token. If there is an existing token that was generated from the same DII and the token has not expired or can be refreshed, the module will use or refresh the existing token instead of generating a new token.
 
-For an example, see [Sample Token](#sample-token).
+As a result, you do not have to check for existing tokens before calling `pbjs.setConfig`. Calling `pbjs.setConfig` when there is an existing token will not have any negative consequences and will ensure that a new token is generated if the existing token can no longer be refreshed.
 
-As long as the refresh token remains valid, the module refreshes the UID2 token as needed.
+The user's DII can be passed to the UID2 module hashed or unhashed. If the DII is passed to the module hashed, it must be normalized before hashing. See [Normalization and Encoding](../getting-started/gs-normalization-encoding.md) for more details.
 
-### Response Storage Options
+The UID2 module will ensure that all DII is hashed and encrypted before sending it to the UID2 service.
 
-When you configure the module to use Client Refresh mode, you must choose **one** of the following options for storing the API response information.
+The following sections demonstrate the different ways to configure the UID2 module and list the requirements for the DII passed to the module.
```

Comment:
should we highlight in bold or add a ":::note" section here to tell pub to pass 1 and only 1 DII (not multiple) in a single setConfig call to UID2 module?

Reply:
Done